### ISSUE TITLE
Doc improvement: subservice details in context base

### ISSUE THREAD

**Issue by fgalan** [Bitergia/fiware-devguide-app#120]
The subservices used by the Tour Guide App in the context data stored at Orion Context Broker should be explained in the documentation. For example, it should be explained that Restaurant entities are located in per-franchise subservices. At least, I have been unable to found that details in existing documentation...

Note that this lack in documentation is causing difficulties among users, as can be seen in the following question at SOF: stackoverflow.com/questions/38351560/fiware-orion-ngsiv2-update-context

Both https://www.fiware.org/devguides/fiware-tour-guide-application-a-tutorial-on-how-to-integrate-the-main-fiware-ges/ and repository documentation should be improved (not sure if the later is generated from the former).

Thanks!

**Comment by albertinisg** [comment]
Agree. There's an [issue](https://github.com/Fiware/tutorials.TourGuide-App/issues/49) to create a readthedocs project just for the TourGuide-App, so we will take this into account. Also, the queries used in that example should be updated for the NGSIv2. 

Thanks for the report!
